src/test-run/commands/index.js

```diff
@@ -187,6 +187,42 @@ export class TypeTextCommand extends Assignable {
     }
 }
 
+export class WaitCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type    = TYPE.wait;
+        this.timeout = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'timeout', type: positiveIntegerArgument, required: true }
+        ];
+    }
+}
+
+export class WaitForElementCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type                       = TYPE.waitForElement;
+        this.selector                   = null;
+        this.elementAvailabilityTimeout = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'selector', type: selector, init: initSelector, required: true },
+            { name: 'elementAvailabilityTimeout', type: positiveIntegerArgument }
```

Comment:
maybe it's just a `timeout` here? I think `elementAvailableTimeout` it's ok for test run option but is redundant for the command. 
`waitForElementCommand(selector, timeout)` looks better than `waitForElementCommand(selector, elementAvailabilityTimeout)`
/cc @inikulin @VasilyStrelyaev